net/table_net_certificate.go

```diff
@@ -27,25 +47,34 @@ func tableNetCertificate(ctx context.Context) *plugin.Table {
 			{Name: "ip_address", Type: proto.ColumnType_IPADDR, Transform: transform.FromField("IPAddress"), Description: "IP address associated with the domain."},
 			{Name: "ip_addresses", Type: proto.ColumnType_JSON, Transform: transform.FromField("IPAddresses"), Description: "Array of IP addresses associated with the domain."},
-			{Name: "is_ca", Type: proto.ColumnType_BOOL, Transform: transform.FromField("IsCA"), Description: "True if the certificate represents a certificate authority."},
+			{Name: "is_ca", Type: proto.ColumnType_BOOL, Transform: transform.FromField("IsCertificateAuthority"), Description: "True if the certificate represents a certificate authority."},
 			{Name: "issuer", Type: proto.ColumnType_STRING, Description: "Issuer of the certificate."},
+			{Name: "issuer_name", Type: proto.ColumnType_STRING, Description: "Issuer of the certificate."},
 			{Name: "issuing_certificate_url", Type: proto.ColumnType_JSON, Transform: transform.FromField("IssuingCertificateURL"), Description: "List of URLs of the issuing certificates."},
 			{Name: "locality", Type: proto.ColumnType_STRING, Description: "Locality of the certificate."},
 			{Name: "not_before", Type: proto.ColumnType_TIMESTAMP, Description: "Time when the certificate is valid from. Also see not_after."},
 			{Name: "organization", Type: proto.ColumnType_STRING, Description: "Organization of the certificate."},
 			{Name: "ou", Type: proto.ColumnType_JSON, Transform: transform.FromField("OU"), Description: "Organizational Unit of the certificate."},
 			{Name: "public_key_algorithm", Type: proto.ColumnType_STRING, Description: "Public key algorithm used by the certificate."},
+			{Name: "public_key_length", Type: proto.ColumnType_INT, Description: ""},
 			{Name: "serial_number", Type: proto.ColumnType_STRING, Description: "Serial number of the certificate."},
 			{Name: "signature_algorithm", Type: proto.ColumnType_STRING, Description: "Signature algorithm of the certificate."},
 			{Name: "state", Type: proto.ColumnType_STRING, Description: "State of the certificate."},
 			{Name: "subject", Type: proto.ColumnType_STRING, Description: "Subject of the certificate."},
+			{Name: "crl_distribution_points", Type: proto.ColumnType_JSON, Transform: transform.FromField("CRLDistributionPoints"), Description: "A CRL distribution point (CDP) is a location on an LDAP directory server or Web server where a CA publishes CRLs."},
+			{Name: "ocsp_server", Type: proto.ColumnType_JSON, Transform: transform.FromField("OCSPServer"), Description: "The Online Certificate Status Protocol (OCSP) is a protocol for determining the status of a digital certificate without requiring Certificate Revocation Lists (CRLs. The revocation check is by an online protocol is timely and does not require fetching large lists of revoked certificate on the client side. This test suite can be used to test OCSP Responder implementations."},
```

Comment:
Do we need both `ocsp_server` and `ocsp` columns? Are we able to combine these in any way, or differentiate their data/intents better?